packages/shared/ReactSymbols.js

```diff
@@ -24,6 +24,9 @@ export const REACT_PORTAL_TYPE = hasSymbol
 export const REACT_FRAGMENT_TYPE = hasSymbol
   ? Symbol.for('react.fragment')
   : 0xeacb;
+export const REACT_USE_STRICT_TYPE = hasSymbol
+  ? Symbol.for('react.use_strict')
```

Comment:
Let's call it the same thing as what we export. `react.strict_mode`

`use strict` is JavaScript.

Reply:
But ... React is JavaScript! 😄